### PR TITLE
Testing practise

### PR DESCRIPTION
**Pull request by braincee**
_**In this pull request, the following requirements were implemented:**_

- My partner and I :
- wrote unit tests for the files operate.js and calculate.js using Jest.
- created unit tests for all React components:
- used Jest snapshots to test the components.
- used React Testing Library to simulate user interaction.
- run my full tests suite using Jest. (All tests should pass.)

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,67 @@
...
+
+  const multiplication = operate(number1, number2, 'x');
+  it('should perform a multiplication', () => {
+    expect(multiplication).toBe('27');
+  });
+
+  const division = operate(number1, number2, '÷');
+  it('should perform a division', () => {
+    expect(division).toBe('3');
+  });
```
Comment (AnselemOdims) [inline src/App.test.js:67]: - [ ] Good job on working on the tests, but please ensure that all  functionalities & scenarios of the `operate.js` and `calculate.js` modules

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @braincee , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.test.js (hunk 1 above)
Comment (braincee) [reply src/App.test.js:66]: Thanks, Anselem for the review comments. my partner and I will implement all functionalities & scenarios of the `operate.js` and `calculate.js`

**Review by Mre55 — APPROVED** [review]
**Hi @braincee  and @MaganaAsiati ,**

# Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

Alt Text


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thank you very much,  @Mre55, @AnselemOdims and @MaganaAsiati 

cf90bb1b741ff44adde054a2e12b82e6

**Pull request merged**